examples/integrated_gradients_transformers.ipynb

```diff
@@ -0,0 +1,1235 @@
+{
```

Comment:
<p>spelling</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/430/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='52'/>

Reply:
The notebook is only a stab with minimal quick comments, I know it must be rewritten and check.  But thanks for the spell checking Janis.